daemon/daemon.go

```diff
@@ -1051,6 +1051,9 @@ func (daemon *Daemon) Shutdown() error {
 // Mount sets container.BaseFS
 // (is it not set coming in? why is it unset?)
 func (daemon *Daemon) Mount(container *container.Container) error {
+	if container.RWLayer == nil {
```

Comment:
+1 for checking for `container.Dead` here, unmount and in Windows, like in `ContainerChanges`

Reply:
Please see the updated version

Reply:
Where?

Reply:
Well, adding a check for `container.Dead` is a change in workflow -- meaning we no longer mount/unmount containers in Dead state. I'm not quite sure if this is the right change....

Reply:
reverted the check for `container.Dead` in mount/umount/change, leaving only in `getInspectData` where a similar check was before
￼